youtube_dl/extractor/bandcamp.py

```diff
@@ -153,8 +154,10 @@ def _real_extract(self, url):
 
 class BandcampAlbumIE(InfoExtractor):
     IE_NAME = 'Bandcamp:album'
-    _VALID_URL = r'https?://(?:(?P<subdomain>[^.]+)\.)?bandcamp\.com(?:/album/(?P<album_id>[^?#]+)|/?(?:$|[?#]))'
-
+    VALID_URLS = \
+    [r'https?://(?:(?P<subdomain>[^.]+)\.)?bandcamp\.com/album/(?:(?P<album_id>[^?#]+)|/?(?:$|[?#]))',
+        r'https?://(?P<subdomain>[^.]+)\.bandcamp\.com/?(?:$|[?#])(?P<album_id>)']
```

Comment:
[See comments on previous revision](https://github.com/rg3/youtube-dl/pull/12758#discussion_r118849945).  This originally started with 79b3f1830328117edf9e301a42fd8cb692eaac0f.

It sounds like instead for overriding `suitable` you want a big regex with an alternation (`|`) which will require renaming the subgroups to be unique and either later on.